### ISSUE TITLE
Handle names better (Old 297)

### ISSUE THREAD

**Issue by NFarrington** [VATSIM-UK/core#70]
In gitlab by @A-Lawrence on Aug 31, 2015, 01:07

Member 1195594 currently has none-ascii characters in his name. 

Annoyingly, it's how it's presented to us from Cert, but it's something we need to consider.

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Aug 31, 2015, 01:12

No longer encode/decode names - as discussed in #488434 (297), resolves #488434 (297)


ÔåÆ <<cset 6a532f5ff31f>>

NFarrington - 2015-03-10T08:08:08.919647+00:00

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Aug 31, 2015, 01:12

@ALawrence66 I'd like to switch this back, and make it so that we don't touch the encoding on the data from Cert.

Given that, for example:


```
#!

https://cert.vatsim.net/vatsimnet/idstatus.php?cid=1195594 - issue with name encoding
https://cert.vatsim.net/vatsimnet/idstatus.php?cid=976512 - no issue with name encoding (decoding breaks the name in our database)
```


Per Kieran's post (http://forums.vatsim.net/viewtopic.php?f=134&t=66626), names should already be properly encoded. Per an old post (http://forums.vatsim.net/viewtopic.php?f=1&t=91), members should also correct their own names.

Finally, incorrectly coded names are also identical on the .net forums: http://forums.vatsim.net/memberlist.php?mode=viewprofile&u=1195594

In summary:
Some data in cert is properly encoded, some isn't. We don't want to break the members who have correct encoding. Members with wrong encoding should contact the membership department. Decoding UTF-8 to UTF-8 also seems a little bizarre.
So, are we good to just accept the data we get from Cert, without encoding modification?

NFarrington - 2015-03-05T10:27:06.384278+00:00

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Aug 31, 2015, 01:12

Stop encoding decoded names. Resolves #488434 (297), again.


ÔåÆ <<cset 7b4a8dd096a4>>

NFarrington - 2015-03-02T21:51:38.054589+00:00

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Aug 31, 2015, 01:12

All names now run through utf8_decode. If this causes any issues, we can apply a new solution.

NFarrington - 2015-02-28T21:45:43.281352+00:00

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Aug 31, 2015, 01:17

mentioned in issue #261

**Comment by NFarrington** [comment]
In gitlab by @A-Lawrence on Aug 31, 2015, 01:17

mentioned in issue #265
